src/pages/KyberDAO/Vote/ProposalItem/index.tsx

```diff
@@ -340,22 +341,36 @@ function ProposalItem({
           </ExpandButton>
         </RowBetween>
         {(show || isActive) && isMobile && (
-          <RowFit gap="8px">
-            <StatusBadged color={tagColor()} onClick={() => onBadgeClick?.(proposal.status)}>
-              {proposal.status}
-            </StatusBadged>
-            <IDBadged>ID #{proposal.proposal_id}</IDBadged>
-          </RowFit>
+          <RowBetween>
+            <RowFit gap="8px" flexWrap="wrap">
+              <StatusBadged color={tagColor()} onClick={() => onBadgeClick?.(proposal.status)}>
+                {proposal.status}
+              </StatusBadged>
+              <IDBadged>ID #{proposal.proposal_id}</IDBadged>
+            </RowFit>
+            {isActive && (
+              <RowFit gap="4px" flexShrink={0}>
+                <Text color={theme.subText} fontSize={12}>
+                  <Trans>Voting ends in: </Trans>
+                </Text>
+                <StatusBadged color={theme.primary}>
+                  {readableTime(proposal.end_timestamp - Date.now() / 1000)}
```

Comment:
add some interval re-render mechanism? E.g useInterval

Reply:
this time only show in minutes, its ok to not re-render too frequent here